docker/requirements.txt

```diff
@@ -1,15 +1,14 @@
 pyjks==20.0.0
-shortuuid==1.0.11
-behavex==2.0.1
-docker==5.0.0
-kafka-python==2.0.2
-confluent-kafka==1.7.0
-PyYAML==6.0.1
-m2crypto==0.41.0
-watchdog==2.1.2
-pyopenssl==23.0.0
-azure-storage-blob==12.13.0
-prometheus-api-client==0.5.0
+shortuuid==1.0.13
+behavex==4.0.10
+docker==7.1.0
+confluent-kafka==2.8.0
+PyYAML==6.0.2
+m2crypto==0.43.0
```

Comment:
0.43.0 doesnt install on macOS Sonoma 14.6.1 arm64 (python3.12)
can we keep 0.41.0?